content/enterprise_features/container.md

```diff
@@ -316,7 +316,87 @@ of the generated profiles in `/headless/.mozilla/firefox/long_random_id.default`
 
 **Attention:** If this property is not set, added certificates will have no effect.
 
-## 7 Overview Environment Variables
+## 7 Remote desktop protocol (RDP) connections {#remote-connection-container}
+There are use cases that cannot be covered with the standard Linux based Sakuli container e.g. testing software that is
+only available for Microsoft Windows or automating processes in SAP. If you still want to leverage the flexibility and
+scalability of a container based Sakuli execution, we provide the `sakuli-remote-connection` container. This image
+ships with a [remmina](https://remmina.org/) installation to connect to a remote Windows machine using RDP.
+
+### 7.1 RDP example
+
+Sakuli check:
+{{<highlight typescript>}}
+(async () => {
+    const remmina = new Application(`remmina --connect <path/to/remmina/config.rdp>`);
```

Comment:
I think we should add a hint which indicates that this file needs to be mounted into the container